templates/html-header.twig

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html {{ fn('language_attributes', 'html') }}>
+<html {{ fn('language_attributes', 'html') }} data-base="{{ constant('WP_HOME') }}">
```

Comment:
The base data attribute is used by the theme editor to put a local storage namespace, otherwise it loads the same settings on each instance on the same domain. This is just the best fix I came up with, maybe I'm overlooking an existing way to get this value on the front end.